coordinated-entry-screening/static/css/custom.css

```diff
@@ -83,8 +76,21 @@ nav {
 }
 
 /* Chatbot */
+.bot-msg, .botui-actions-buttons, .botui-message-content, .card-header {
+    font-size: 16px;
+}
+
+.bot-msg div {
+    padding-bottom: 4px;
+}
+
+.botui-message-content {
+    line-height: 1.4;
+    max-width: 70%;
```

Comment:
We should add a media query here for small screens; the text becomes unreadable.
<img width="373" alt="screen shot 2019-01-09 at 10 29 27 am" src="https://user-images.githubusercontent.com/6961258/50913526-2bda7680-13fa-11e9-8914-a12e140a2223.png">
<img width="312" alt="screen shot 2019-01-09 at 10 29 39 am" src="https://user-images.githubusercontent.com/6961258/50913527-2c730d00-13fa-11e9-9966-c5b119cc14d5.png">

Reply:
I am not sure the best solution. I increased the size of the content box (for smaller devices). Does that make sense?

Reply:
Yes! I would also scale B.E.N.'s avatar, lower the font size, and possibly lower the line height. The latter two might be easier if we define them via rem values (which might eliminate the need for a media query, I'm not totally sure).

Articles:
[rem values in-depth](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/)
[much more concise breakdown](https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97)